docs/keycloak.md

```diff
@@ -0,0 +1,42 @@
+# Configuring Keycloak for Gatekeeper
+
+## Prerequisites
+- You need access to a Keycloak instance and to configure a realm.
+
+## TL;DR:
+- Create a client and set its "Access Type" to "Confidential". The name of the client is the CLIENT_ID and the CLIENT_SECRET is in the credentials tab of the client.
+- Configure the client with your <gatekeeper url>/callback as "Valid Redirect URIs" and your frontend URL as "Web Origins"
+- The DISCOVERY_URL is the endpoint "OpenID Endpoint Configuration" found on the "Realm Settings" page. (Right click and copy link)
+
+### Create and configure the Client
+
+#### Create the client
+1. Log in to Keycloak and select your realm.

```

Comment:
Github markdown lists supports lists being written as
```
1. Hey
1. Ho
1. Britney
```
"Proof": 

1. Hey
1. Ho
1. Britney

Then you don't need to re-write numbers if you change orders or whatnot.

Reply:
That totally f's up my OCD